plugin.video.infowars/LICENSE.txt

```diff
@@ -1,339 +0,0 @@
-                    GNU GENERAL PUBLIC LICENSE
-                       Version 2, June 1991

```

Comment:
I don't think my windows pc liked that there was two license.txt files in here and it won't let me add it back. Ideas?

Reply:
Do not use `git add -- .` to stage your changes.

Reply:
Ok, will do. Thanks @Rechi